src/tests/unit/directory-serializer.test.js

```diff
@@ -0,0 +1,34 @@
+import chai from 'chai';
```

Comment:
You should take a look at some of the test helpers [here](https://github.com/osu-mist/game-curator-api/blob/master/tests/unit/test-helpers.js#L45). Almost any change to the swagger document will break your tests. You should be able to copy over those helpers without making too many changes.